db/create_tables.php

```diff
@@ -7,6 +7,8 @@
 require_once($cwd[__FILE__] . "/my_query.php");
 
 $drop_tables = false;
```

Comment:
Just double checking, these booleans and their use in the conditionals are the only changes to this, right?